sonaranalyzer-dotnet/rspec/vbnet/S1313_vb.net.html

```diff
@@ -28,9 +33,13 @@ <h2>Compliant Solution</h2>
 Dim address = IPAddress.Parse(ip)
 </pre>
 <h2>Exceptions</h2>
+<p>No issue is reported for the following cases because they are not considered sensitive:</p>
 <ul>
-  <li> Although "::" is a valid IPv6 address, the rule doesn't report on it. </li>
-  <li> No issue is reported for 127.0.0.1 because loopback is not considered as sensitive </li>
+  <li> Loopback addresses 127.0.0.0/8 in CIDR notation (from 127.0.0.0 to 127.255.255.255) </li>
+  <li> Broadcast address 255.255.255.255 </li>
```

Comment:
I could not find any checks for "255.255.255.255" or "0.0.0.0". I think we need to add a jira ticket for that.

@eric-therond-sonarsource: do we have an MMF for this?

Reply:
I just created this ticket to fix the gap: https://github.com/SonarSource/sonar-dotnet/issues/3450